subsystem/kubeapi_test.go

```diff
@@ -1124,6 +1124,43 @@ var _ = Describe("[kube-api]cluster installation", func() {
 		Expect(cluster.ImageGenerated).Should(Equal(true))
 	})
 
+	It("deploy infraEnv in default namespace before clusterDeployment", func() {
+		// Deploy InfraEnv in default namespace
+		err := kubeClient.Create(ctx, &v1beta1.InfraEnv{
```

Comment:
the cleanup will not remove from a default namespace, please add a defer that remove this after this test is over.

Reply:
right, added.